src/main/frontend/fs/capacitor_fs.cljs

```diff
@@ -159,6 +158,19 @@
                       (error-handler error)
                       (log/error :write-file-failed error)))))))))
 
+(defn- get-file-path [dir path]
+  (let [[dir path'] (map #(-> (when %
```

Comment:
no need to introduce `path'`

Reply:
`path'` is used to store a modified path, which is used below.